src/Command/StatusCommand.php

```diff
@@ -27,9 +29,6 @@ class StatusCommand extends Command
 --daemonize option.
 EOH;
 
-    /** @var null|string Cannot be defined explicitly due to parent class */
-    public static $defaultName = 'mezzio:swoole:status';
```

Comment:
I see this as a breaking change, BTW :thinking:

Reply:
Yeah, it sure is. Maybe we could keep the properties even though they're unused in this repository.